src/app/styles/login.tsx

```diff
@@ -0,0 +1,13 @@
+import styled from 'styled-components';
```

Comment:
You should use 'styled-components/native' import here, then you should use styled.View and styled.Text below.